test/Azure.Functions.Cli.Tests/PublishActionTests.cs

```diff
@@ -14,38 +14,26 @@ public class PublishActionTests
     {
         TestAzureHelperService _helperService = new TestAzureHelperService();
 
-        [Fact]
-        public async Task NetFrameworkVersion_DotnetIsolated_Linux_Consumption_AlreadyEmpty()
-        {
-            var site = new Site("test")
-            {
-                Kind = "linux",
-                Sku = "dynamic",
-                LinuxFxVersion = null
-            };
-
-            await PublishFunctionAppAction.UpdateFrameworkVersions(site, WorkerRuntime.dotnetIsolated, "6.0", false, _helperService);
-
-            // no-op if already null or empty
-            Assert.Null(_helperService.UpdatedSettings);
-        }
-
-        [Fact]
-        public async Task NetFrameworkVersion_DotnetIsolated_Linux_Consumption_Updated()
+        [Theory]
+        [InlineData(null, "6.0")]
```

Comment:
Also use `net5.0` as input

Reply:
Resolved, as we do not plan on supporting .NET 5.0 scenarios with Core Tools V4